CONTRIBUTING.md

```diff
@@ -120,3 +120,13 @@ cd ReSwift-gh-pages
 # Documentation is ready to be committed.
 ```
 
+## Pull Request Technicalities
+
+Here are some hard facts for reference:
+
+* You're always welcome to open Pull Requests, even if the result is not quite finished or you need feedback!
+* Run Swift Lint locally to ensure there are no code-level oddities.
```

Comment:
Also, I would rephrase to something like this:

`Run SwiftLint locally to ensure your code-style is consistent with the rest of the codebase`.